src/resauce/core.clj

```diff
@@ -1,7 +1,7 @@
 (ns resauce.core
   (:require [clojure.java.io :as io])
   (:import [java.io File]
-           [java.net JarURLConnection URI URL]
+           [java.net JarURLConnection URL URI]
```

Comment:
This change looks like it's superfluous now.

Reply:
Right, missed that.